VERSION

```diff
@@ -0,0 +1 @@
+201602
```

Comment:
Yes, that's the recommended version format for salt formulas. https://docs.saltstack.com/en/latest/topics/spm/index.html#version